### DISCUSSION TITLE
Terabox Cookie

### DISCUSSION THREAD

**Discussion by goldbugs** [AlistGo/alist#2966] [Q&A]
I tried to set up Terabox in Alist, but could not find the cookie that works.
Using Chrome, I logged on Terabox, then F12>Developer Tool>Application>Cookies>Terabox, I tried all (>10) cookies, and none worked.

How exactly can I find the cookie?
Thanks

**Comment by wotime** [comment]
没嗨用，国内下载处处闲置，得挂梯下，中转也不行，成功挂载上了实测。

**Reply by goldbugs** [reply]
啊 成功了
应该F12>Developer Tool>network, 参照“夸克网盘”，把所有cookies连屎带尿，通通填入，而不是单个cookie。

**Comment by vintue** [comment]
遇到同样的问题，一样把整个cookie值复制过去，夸克的就可以，TeraBox就不行

**Reply by goldbugs** [reply]
> 遇到同样的问题，一样把整个cookie值复制过去，夸克的就可以，TeraBox就不行

我把整个cookie值复制到TeraBox就行
